lib/datadog/di/configuration/settings.rb

```diff
@@ -188,6 +188,20 @@ def self.add_settings!(base)
                   o.type :bool
                   o.default false
                 end
+
+                # Enable logging of dynamic instrumentation activity.
+                # This is quite verbose.
+                option :trace_logging do |o|
```

Comment:
As a question, since you want to have DI specific setting, does it makes sense to rename it to DI specific name even if internally you rely on another ENV variable. Because why `trace` should be in the name?